servers/curated-corpus-api/src/admin/resolvers/queries/Section/getSectionWithSectionItems.integration.ts

```diff
@@ -286,6 +286,10 @@ describe('queries: Section (getSectionsWithSectionItems)', () => {
 
   it('should return custom section fields', async () => {
     // Create a custom section with description, heroTitle, and heroDescription
+    // Use yesterday's date to ensure the section is LIVE (not SCHEDULED)
+    const yesterday = new Date();
+    yesterday.setDate(yesterday.getDate() - 1);
```

Comment:
interesting - was this causing a flaky test?

Reply:
Yes, it failed when PT and ET were on different dates.